zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/utils/SearchEngineClientUtils.java

```diff
@@ -41,19 +43,20 @@ public static InputStream appendToFileSchemaSettings(
     indexBlock.put("number_of_shards", settingsToAppend.getNumberOfShards());
     indexBlock.put("number_of_replicas", settingsToAppend.getNumberOfReplicas());
 
-    return new ByteArrayInputStream(MAPPER.writeValueAsBytes(map));
+    return new ByteArrayInputStream(objectMapper.writeValueAsBytes(map));
   }
 
-  public static String listIndices(final List<IndexDescriptor> indexDescriptors) {
+  public String listIndices(final List<IndexDescriptor> indexDescriptors) {
     return indexDescriptors.stream()
         .map(IndexDescriptor::getFullQualifiedName)
         .collect(Collectors.joining(","));
   }
 
-  public static <T> T mapToSettings(
+  public <T> T mapToSettings(
       final Map<String, String> settingsMap, final Function<InputStream, T> settingsDeserializer) {
     try (final var settingsStream =
-        IOUtils.toInputStream(MAPPER.writeValueAsString(settingsMap), StandardCharsets.UTF_8)) {
+        IOUtils.toInputStream(
+            objectMapper.writeValueAsString(settingsMap), StandardCharsets.UTF_8)) {
```

Comment:
❓ Why we are not using here the writer/readers in this class?

Reply:
I don't remember, but there a case when a method was not present in the reader or writer, but was present in the object mapper.  I need to double check

Reply:
I checked, `ObjectReader` is missing the overload `readValue(String, TypedReference)`